modoboa_stats/handlers.py

```diff
@@ -25,5 +26,6 @@ def menu(sender, location, user, **kwargs):
 @receiver(signals.get_graph_sets)
 def get_default_graphic_sets(sender, **kwargs):
     """Return graphic set."""
-    gset = graphics.MailTraffic()
+    gset = graphics.MailTraffic(param_tools.get_global_parameter(
+            "greylist", app="modoboa_stats"))
```

Comment:
You don't have to specify the app in this case, it will be guessed automatically.